httpx/_types.py

```diff
@@ -41,6 +41,8 @@
     "Headers",
     Dict[str, str],
     Dict[bytes, bytes],
+    Dict[str, Optional[str]],
+    Dict[bytes, Optional[bytes]],
     Sequence[Tuple[str, str]],
     Sequence[Tuple[bytes, bytes]],
```

Comment:
I _think_ (?) we also need to update these:

```suggestion
    Sequence[Tuple[str, Optional[str]]],
    Sequence[Tuple[bytes, Optional[bytes]]],
```

Reply:
Adding this. Also added a check under model layer to ignore None values for Sequence.